frontend/test/metabase/scenarios/collections/personal-collections.cy.spec.js

```diff
@@ -73,6 +74,18 @@ describe("personal collections", () => {
       });
     });
 
+    it("should not be able to move or archive a personal collection", () => {
+      cy.visit("/collection/root");
+
+      openEllipsisMenuFor(`${getFullName(USERS.admin)}'s Personal Collection`);
+
+      popover().within(() => {
+        cy.findByText("Bookmark").should("be.visible");
+        cy.findByText("Move").should("not.exist");
+        cy.findByText("Archive").should("not.exist");
+      });
+    });
+
```

Comment:
Adding a step to test that we can't use select these personal collections for bulk actions could be necessary https://github.com/metabase/metabase/pull/28229#issuecomment-1429704024